spec/presenters/root_topic_presenter_spec.rb

```diff
@@ -43,4 +31,18 @@
       expect(rendered[:public_updated_at]).to eq(page_2.updated_at.iso8601)
     end
   end
+
+  describe 'render_links_for_publishing_api' do
```

Comment:
`describe '#render_links_for_publishing_api'` will make it clear that this is a instance method.